precheck.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+sbt clean scalafmt test:scalafmt it:scalafmt coverage test it/Test/scalafmt coverageReport scalafmtSbt
```

Comment:
A couple things regarding this script:
1. `it:scalafmt` does not do anything due to project configuration. You should use `it/Test/scalafmt` instead.
2. Integration tests are not running with this script.

Additionally, I don't think we've been adding `scalafmtSbt` to this script in other repositories. But there is no harm to leaving it.